app/models/concerns/case_states.rb

```diff
@@ -81,11 +81,15 @@ def respond(current_user)
   end
 
   def close(current_user)
-    state_machine.close!(acting_user: current_user, acting_team: managing_team)
+    state_machine.close!(acting_user: current_user, acting_team: find_team(current_user))
```

Comment:
By using `find_team(current_user)` here you're anticipating the case may be closed by the responder, but that's only true for SAR cases, right? FOI cases shouldn't ever be closed by a responder.